CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ Allowed headings:
 
 ## [Unreleased]
 
+### Added
+
+* `WarningView` has new attribute `defaultPadding` which removes the default padding from the component when set to false. When not set, component has `8dp` padding.
```

Comment:
```suggestion
* `WarningView` has new attribute `defaultPadding` which removes the default padding from the component when set to false. When this is true, component has `8dp` padding.
```

Reply:
^ or similar. Basically the statement isn't correct anymore so needs updating

Reply:
Have updated to clarify - but we do not need to set this explicitly to enable the default padding, it only needs to be set in the xml if we want to remove the padding. (This is a bit against the way we usually work - define if "true" - but this is to prevent affecting any existing usages of the component)